Nos/Views/New Note/NewNoteView.swift

```diff
@@ -209,7 +209,7 @@ struct NewNoteView: View {
         }
         
         do {
-            var (content, tags) = NoteParser.parse(attributedText: text.attributedString)
+            var (content, tags) = NoteParser().parse(attributedText: text.attributedString)
```

Comment:
Again, I'd prefer this dependency on NoteParser to be declared up top as a `@Dependency` property rather than instantiated here inside a func.

Reply:
I added a dependency just in the same way we added for URLParser